docs/grammar.md

```diff
@@ -9,14 +9,14 @@ The basic building blocks of the textX language are rules. Each rule is written
 in the following form:
 
     Hello:
-      'hello' who=ID;
+      'hello' who=ID
     ;
 
-This rule is called `Hello`. After the name is a colon. Between the colon and
-the semicolon at the end is a body of the rule given as a textX expression. This
-rule tells us that the pattern of `Hello` objects in the input string consists
-of the word `hello`, followed by the ID rule (here ID is a rule reference to the
-built-in rule, more about this in a moment).
+This rule is called `Hello`. After the rule name, there is a colon. The body of the
+rule is given as a textX expression, starting at the colon and and ending with a
```

Comment:
Thanks for the contribution! Looks good. There is just a minor glitch at this line -- double *and*.